macros/fivetran/base/fivetran_fb_ads.sql

```diff
@@ -0,0 +1,46 @@
+{% macro fivetran_fb_ads() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ads') }}
+
+{% endmacro %}
+
+--there are multiple records for different statuses, but we don't need them for our purposes yet.
```

Comment:
if we have multiple rows for different statuses, we should be using `last_value()` window functions to get the correct / current value of the row / status...